### PR TITLE
Regression test add solution services

### PR DESCRIPTION
**Pull request by imranbashir01**

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            CommonActions.ClickSave();
+
+            CommonActions.PageLoadedCorrectGetIndex(
+                typeof(AdditionalServicesController),
+                nameof(AdditionalServicesController.EditAdditionalService))
+                .Should()
+                .BeTrue();
+        }
+
+        public void AddCapbilities()
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs]: Spelling: `Capbilities` -> `Capabilities`

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        {
+            ListPricesForServices = new ListPricesForServices(driver, commonActions);
+            Factory = factory;
+        }
+
+        public ListPricesForServices ListPricesForServices { get; }
+
+        public LocalWebApplicationFactory Factory { get; }
+
+        public void AddAdditionlService(string solutionId, string priceType)
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs]: Spelling: `Additionl` -> `Additional`

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                typeof(AdditionalServicesController),
+                nameof(AdditionalServicesController.AddAdditionalService))
+                .Should().BeTrue();
+
+            AddAdditionalServiceDetails();
+            AddCapbilities();
+            AddListPrices(priceType, service);
+        }
+
+        public void AddAdditionalServiceDetails()
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs]: Subsequent moethods could be marked as private as they're currently only used by `AddAdditionalService` on line 35

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAssociatedServices/SolutionAssociatedService.cs [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            CommonActions.PageLoadedCorrectGetIndex(
+                typeof(AssociatedServicesController),
+                nameof(AssociatedServicesController.AddAssociatedService))
+                .Should().BeTrue();
+
+            AddAssociatedServiceDetails();
+            AddListPrices(priceType, service);
+        }
+
+        public void AddAssociatedServiceDetails()
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAssociatedServices/SolutionAssociatedService.cs]: Subsequent methods could be private as they're used only by `AddAssociatedService`

**Review by bjss-colin — CHANGES_REQUESTED** [review]

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs (hunk 2 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs]: Good spot

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs (hunk 3 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAdditionalServices/SolutionAdditionalService.cs]: changed as advised

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAssociatedServices/SolutionAssociatedService.cs (hunk 4 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Admin/ManageSolutions/SolutionAssociatedServices/SolutionAssociatedService.cs]: changed as advised

**Review by bjss-colin — APPROVED** [review]

**Pull request merged**